src/addressResolvers/index.ts

```diff
@@ -55,7 +56,12 @@ async function _call(fnName: string, input: string[], maxInputLength: number) {
       );
 
       return Object.fromEntries(
-        _input.map(item => [item, results.map(r => r[item]).filter(i => !!i)[0] || ''])
+        _input.map(item => [
+          item,
+          results
+            .map(r => (item === EMPTY_ADDRESS && fnName === 'lookupAddresses' ? '' : r[item]))
```

Comment:
Maybe we can filter empty address in withoutEmptyValues?

Reply:
I would prefer to keep `withoutEmptyValues` as a function to remove empty values from the object, as the name implied.

I added a new function which only purpose is to remove empty address keys from object